lib/scutil/host-fp-folding.c

```diff
@@ -24,7 +24,7 @@
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
-#ifdef _WIN64
+#if defined(_WIN64) && !defined(_M_ARM64)
```

Comment:
Why? From https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

> _WIN64 Defined as 1 when the compilation target is 64-bit ARM or x64. Otherwise, undefined.